mender-auth/cli/actions.hpp

```diff
@@ -50,12 +50,26 @@ using ExpectedActionPtr = expected::expected<ActionPtr, error::Error>;
 
 class DaemonAction : virtual public Action {
 public:
-	static ExpectedActionPtr Create(const conf::MenderConfig &config, const string &passphrase);
+	static ExpectedActionPtr Create(
+		const conf::MenderConfig &config, const string &passphrase, bool force_bootstrap);
```

Comment:
I wouldn't mind a `const` bool, but feel free to ignore (just for readability) :)

Reply:
I actually had the `const bool` first but somehow didn't look nice on my eyes. I'll switch back to it now!